### DISCUSSION TITLE
working with hacktribe on electribe 2 music production station

### DISCUSSION THREAD

**Discussion by Zahni666** [bangcorrupt/hacktribe#332] [Q&A]
hello guys hopefully you can help me little bit. first of all, hope you all good in health and your families :)

I got an Korg electribe 2 music production station, I have hack tribe firmware installed and tried to load my own samples, but aim stuck at this point... after some research I saw that I have to get the original firmware and the original all sample file and edit them to get my own samples, is this true and how ham doing this? I tried to go back to original firmware but without success, seems ham doing something wrong, hopefully you can help me please

cheers

**Comment by bangcorrupt** [comment]
[FAQ#i-cant-import-my-samples](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#i-cant-import-my-samples)

[FAQ#how-can-i-restore-the-factory-firmware](https://github.com/bangcorrupt/hacktribe/wiki/FAQ#how-can-i-restore-the-factory-firmware)
